website/docs/r/organization_token.html.markdown

```diff
@@ -24,7 +24,7 @@ resource "tfe_organization_token" "test" {
 
 The following arguments are supported:
 
-* `organization` - (Required) Name of the organization.
+* `organization` - (Optional) Name of the organization. If omitted, organization provider config must be defined.
```

Comment:
I was re-reading this and wondered if it is clearer to say: `If omitted, organization must be defined in the provider config.` WDYT?